setup/artifacts/github.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title:  "Configuring GitHub Webhooks"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker can be configured to listen to changes to a repository in
+[GitHub](https://github.com). These steps will take you through how to
+configure a GitHub artifact account to be able to download
+
+## Prerequisites
+
+* You need a [GitHub](https://github.com) account to authenticate as.
```

Comment:
Can you delete "...to authenticate as"? Just say you need a GitHub account and assume the user understands she'll need to authenticate?